src/clusterfuzz/_internal/platforms/android/settings.py

```diff
@@ -153,6 +153,15 @@ def get_sanitizer_tool_name():
   return None
 
 
+def get_memory_tool_name():
+  """Return memory tool name."""
```

Comment:
Why doesn't this return a result for sanitizers? Maybe let's name it something mte specific?

Reply:
There is a separate function for sanitizers (`get_sanitizer_tool_name()`).

I update the function name to be more indicative of MTE